src/commands/apphosting-rollouts-create.ts

```diff
@@ -1,27 +1,116 @@
 import * as apphosting from "../gcp/apphosting";
-import { logger } from "../logger";
 import { Command } from "../command";
 import { Options } from "../options";
 import { needProjectId } from "../projectUtils";
+import { FirebaseError } from "../error";
+import * as ora from "ora";
+import { getRepoDetailsFromBackend, listAllBranches } from "../gcp/devConnect";
+import {
+  getGitHubBranch,
+  getGitHubCommit,
+  promptGitHubBranch,
+} from "../apphosting/githubConnections";
+import { orchestrateRollout } from "../apphosting";
+import { confirm } from "../prompt";
+import { logBullet } from "../utils";
+import { consoleOrigin } from "../api";
 
-export const command = new Command("apphosting:rollouts:create <backendId> <buildId>")
+export const command = new Command("apphosting:rollouts:create <backendId>")
   .description("create a rollout using a build for an App Hosting backend")
   .option("-l, --location <location>", "specify the region of the backend", "us-central1")
   .option("-i, --id <rolloutId>", "id of the rollout (defaults to autogenerating a random id)", "")
+  .option("-b, --git-branch <gitBranch>", "repository branch to deploy (defaults to 'main')", "")
+  .option("-c, --commit <commit>", "git commit to deploy (defaults to the latest commit)", "")
+  .option("-f, --force", "skip confirmation")
```

Comment:
I believe there's a withForce method that you can use here instead of a custom force flag